values.yaml

```diff
@@ -120,4 +106,29 @@ relay:
 
   ## Secrets don't like '-'
   secrets:
-    relayCogToken: 
+    relayCogToken:
+
+nginx:
+  name: nginx
+  replicaCount: 1
+  image:
+    repository: nginx
+    tag: 1.11
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  serviceType: LoadBalancer
+  ingress:
```

Comment:
hmm I wonder if we should change it to only use Nginx if the user specifies that they are running on GKE. The ingress would be under Cog, but the file would change where it points depending on the provider.